packages/kbn-es/src/cluster.js

```diff
@@ -85,9 +85,16 @@ exports.Cluster = class Cluster {
     this._log.info(chalk.bold('Installing from snapshot'));
     this._log.indent(4);
 
+    const version = process.env.SNAPSHOT_VERSION || options.version;
+
+    if (version !== options.version) {
+      this._log.info(`Using Elasticearch version ${version} from env.SNAPSHOT_VERSION`);
```

Comment:
What was the reason this was needed? When running 6.7 against 7.0 we won't be able to make this change since we don't know what version we are running against.